modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterInitializer.java

```diff
@@ -50,11 +55,20 @@ public class ClusterInitializer {
 
     private final ClusterService clusterService;
 
+    private final ConfigurationDefaultsSetter configurationDefaultsSetter;
+
+    private final ConfigurationValidator clusterConfigurationValidator;
     private final CmgMessagesFactory msgFactory = new CmgMessagesFactory();
 
     /** Constructor. */
-    public ClusterInitializer(ClusterService clusterService) {
+    public ClusterInitializer(
+            ClusterService clusterService,
+            ConfigurationDefaultsSetter configurationDefaultsSetter,
+            ConfigurationValidator clusterConfigurationValidator
```

Comment:
I don't see reasons to validate configuration here one more time. An explanation in the comment somewhere would be helpful. Maybe we should remove it